Assets/Code/Sprite/AnimatedSprite/Character/Character.cs

```diff
@@ -80,53 +63,18 @@ IEnumerator MoveIn()
         StartCoroutine(Idle());
     }
 
-    private void PlayMovementAnimation()
+    protected void PlayMovementAnimation()
     {
         StopAllCoroutines();
         StartCoroutine(MoveOut());
-    } 
+    }
 
     // Start is called before the first frame update
```

Comment:
If we are going to change the Start() call to an Awake() call, we should update the comment above it... Or just delete the comment, because it is just telling us that the Start() function starts things, and the Awake() function (presumably) awakens things.